services/bettersocial/user/BetterSocialUnblockUserV2.js

```diff
@@ -43,6 +68,8 @@ const BetterSocialUnblockUserV2 = async (token, selfUserId, targetUserId) => {
   }
 
   BetterSocialScoreUnblockUser(selfUserId, targetUserId);
+  if (anonymousBlockFromSign)
+    BetterSocialScoreUnblockUser(selfUserId, targetAnonymousUserId?.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This part of the code assumes that `targetAnonymousUserId` is not null, but there's no guarantee of this. If `targetAnonymousUserId` is null, this could cause an error. Consider adding a null check before calling `BetterSocialScoreUnblockUser`.
```diff
   BetterSocialScoreUnblockUser(selfUserId, targetUserId);
+  if (targetAnonymousUserId)
     if (anonymousBlockFromSign)
       BetterSocialScoreUnblockUser(selfUserId, targetAnonymousUserId?.user_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->